site/content/blog/2022-07-01-whats-new-in-svelte-july-2022.md

```diff
@@ -0,0 +1,97 @@
+- Steaming in endpoints is now supported ([#3419](https://github.com/sveltejs/kit/issues/3419)). This was enabled by switching to the Undici `fetch` implementation ([#5117](https://github.com/sveltejs/kit/pull/5117))
+- Static assets can now be symlinked in development environments ([#5089](https://github.com/sveltejs/kit/pull/5089))
+- `server` and `prod` environment variables are now available as a correlary to `browser` and `dev` ([#5251](https://github.com/sveltejs/kit/pull/5251))
+
+---
+
+## Community Showcase
+
+**Apps & Sites built with Svelte**
+- [Virtual Maker](https://www.virtualmaker.net/) lets you make interactive 3D and VR scenes in your browser
+- [Apple Beta Music](https://www.reddit.com/r/sveltejs/comments/v7ic2s/apple_beta_music_uses_svelte/) appears to have been written in some combination of Svelte and web components
+- [Itatiaia](https://www.itatiaia.com.br/), the largest radio station in the country of Brazil just relaunched its news portal in SvelteKit
+- [Pronauns](https://www.pronauns.com) helps you learn pronunciation online with IPA to speak better and sound more native
+- [Immich](https://www.immich.app/) is an open source, high performance self-hosted backup solution for videos and photos on your mobile phone
+- [Pendek](https://github.com/leovoon/link-shortener) is a link shortener built with SvelteKit, Prisma and PlanetScale
+- [Grunfy](https://grunfy.com/tools) is a set of guitar tools - recently migrated to SvelteKit
+- [Radiant: The Future of Radio](https://play.google.com/store/apps/details?id=co.broadcastapp.Radiant) is a personal radio station app built with Svelte and Capacitor
+- [Imperfect Reminders](https://imperfectreminders.mildlyupset.com/) is a todo list for things that are only sort of time sensitive
+- [Svelvet](https://github.com/open-source-labs/Svelvet) is a lightweight Svelte component library for building interactive node-based diagrams
+- [publint](https://github.com/bluwy/publint) lints for packaging errors to ensure compatibility across environments
+- [Playlistr](https://github.com/alextana/spotify-playlist-creator) helps manage and create Spotify playlists
+- [Geoff Rich's page transitions demo](https://twitter.com/geoffrich_/status/1534980702785003520) shows how SvelteKit's `beforeNavigate`/`afterNavigate` hooks can make smooth document transitions in the latest Chrome Canary
+- [Menger Sponge](https://twitter.com/a_warnes/status/1536215896078811137) is a fractal built with Threlte
+
+Want to contribute to a site using the latest SvelteKit features? [Help build the Svelte Society site](https://github.com/svelte-society/sveltesociety.dev/issues)!
+
+
+**Learning Resources**
+
+_Featuring Svelte Society_
```

Comment:
I wonder if there's a better heading for this. E.g. the Full Stack Documentation talk isn't really featuring Svelte Society. Maybe something like "Created by the Svelte team" (the documentary is primarily created by OfferZen, but the Svelte Team certainly played a role in its creation)

Reply:
"Starring the Svelte team"?